site/layouts/partials/head.html

```diff
@@ -24,3 +24,4 @@
     <script src="{{ .Site.BaseURL }}/js/tabs.js"></script>
     <script src="{{ .Site.BaseURL }}/js/main.js"></script>
     <script async defer src="https://buttons.github.io/buttons.js"></script>
+    <script type="text/javascript" src="//cdn.bizible.com/scripts/bizible.js" async=""></script>

```

Comment:
The MIME type `text/javascript` is obsolete (https://tools.ietf.org/html/rfc4329#section-7.1). Could just remove that attribute.

I would also be a fan of using the minimized attribute form for async like the line above it.